coc-committee-jurisdiction-policy-proposed.md

```diff
@@ -57,7 +57,7 @@ An involved party (a reporter, a person who was targeted or harmed by the incide
 
 * The Project-level CoC responders lacked jurisdiction and were required to transfer the incident to the CNCF CoC Committee per “Transfer by Project CoC Responders” above; 
 * Individuals who have hard conflicts of interest (see the [Conflict of Interest Policy](https://www.cncf.io/conduct/procedures/#conflicts-of-interest)) decided the outcome of the incident;
-* The Project-level responders did not conduct a fair evaluation of the incident (e.g., they failed to review relevant evidence that was available to them);
+* The Project-level responders did not conduct a fair evaluation of the incident (e.g., they failed to consider relevant evidence that was available to them, or the accused was not provided with an opportunity to be heard or present evidence and providing such an opportunity was reasonably likely to result in a different outcome);
```

Comment:
could we drop `and providing such an opportunity was reasonably likely to result in a different outcome`?

Reply:
I actually think this is valuable language.

If we have a hypothetical situation where there is enough evidence (say for example, clear video evidence that provides full context), then the accused person presenting additional evidence that isn't likely to result in a different income could be used to abuse the CoCC process.